p2-maven-plugin/src/main/java/org/eclipse/tycho/p2maven/repository/P2RepositoryKind.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2025, 2025 Hannes Wellmann and others.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Hannes Wellmann - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.tycho.p2maven.repository;
+
+public enum P2RepositoryKind {
+    /** Kind representing a repository with both, metadata and artifacts. */
+    both,
```

Comment:
Just one thing that came into my mind would be possibly name it better `colocated`? At least that is how it is named in case the same url is used for artifact and metadata

Reply:
At least in P2's RepositoryDescriptor class this type/kind is named `both` if `artifact` and `metadata` are at the same location.
I cannot tell how public P2 API names this case (if it's named there at all) or how the doc names it.
Have to do some research on that.